powerauth-webflow/src/main/resources/application.properties

```diff
@@ -43,4 +43,9 @@ spring.jpa.properties.hibernate.connection.useUnicode=true
 #spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults=false
 
 # Hibernate Configuration
-spring.jpa.hibernate.ddl-auto=create
\ No newline at end of file
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
... not related to this change, but we probably should place "none" here. The reason we are not randomly deleting application data is that we override this variable in Docker image setup.